apps/bors_frontend/web/command.ex

```diff
@@ -107,20 +107,19 @@ defmodule BorsNG.Command do
   def parse(comment) do
     comment
     |> String.splitter("\n")
-    |> Enum.flat_map(fn
-      @command_trigger <> " " <> cmd ->
-        trim_and_parse_cmd(cmd)
-      @command_trigger <> ": " <> cmd ->
-        trim_and_parse_cmd(cmd)
-      _ -> []
+    |> Enum.flat_map(fn(string) ->
+      trim_and_parse_cmd(Regex.named_captures(regex(), string))
     end)
   end
 
-  def trim_and_parse_cmd(cmd) do
+  def regex, do: ~r/^#{@command_trigger}:?\s(?<command>.+)/i
```

Comment:
elixir doesn't offer a way to compile regex, right?

Reply:
I think [this *is* the way to compile regex](https://hexdocs.pm/elixir/Regex.html).

> Regular expressions created via sigils are pre-compiled and stored in the `.beam` file.